Bigtable/tests/System/FilterTest.php

```diff
@@ -0,0 +1,2659 @@
+<?php
+/**
+ * Copyright 2018 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Bigtable\Tests\System;
+
+use Google\Cloud\Bigtable\Tests\System\DataClientTest;
+
+/**
+ * @group bigtable
+ * @group bigtabledata
+ */
+class FilterTest extends DataClientTest
+{
+    protected static $insertRows;
+    protected static $expectedRows = [];
+    public static function setUpBeforeClass()
+    {
+        parent::setUpBeforeClass();
+        self::$insertRows = [
```

Comment:
Could we build this out programmatically, or at the very least store the information in a separate file (ex. https://github.com/googleapis/google-cloud-php/blob/master/BigQuery/tests/System/data/table-data.json) and import it?